src/search/pdbs/utils.cc

```diff
@@ -28,6 +38,36 @@ int compute_total_pdb_size(
     return size;
 }
 
+vector<FactPair> get_goals_in_random_order(
+    const TaskProxy &task_proxy, const shared_ptr<utils::RandomNumberGenerator> &rng) {
+    vector<FactPair> goals;
+    for (FactProxy goal : task_proxy.get_goals()) {
+        goals.push_back(goal.get_pair());
+    }
```

Comment:
This could use `get_fact_pairs(task_proxy.get_goals())` from task_properties.h, which also makes sure the vector has the minimum reserved size.